espnet2/asr/frontend/adapters_utils/add_adapters.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+import numpy as np
+
+from espnet2.asr.frontend.adapters_utils.adapters.adapter_transformer import (
+    AdapterTransformerSentenceEncoderLayer,
+)
+from espnet2.torch_utils.model_summary import model_summary
+
+
+def add_adapters_wav2vec2(wav2vec2_model, adapter_down_dim, adapt_layers=[]):
+    """
+    add adapters to wav2vec2 model.
+    * adapter_down_dim - down-projection dimension of adapter.
+    * adapt_layers - list of indices of layers to insert adapters.
+    If `None`, adapters are inserted to every layer. (default=`None`).
+    """
+    if adapt_layers == []:
+        logging.info(
+            "adapt_layers is an empty list. \
+                     No adapters will be inserted."
+        )
+
+        return
+
+    # freeze all layers
```

Comment:
A minor suggestion: is it cleaner / easier to understand if you merge this and the following parameter-freezing lines?